File: C#/Practice/Mother of A 3-year-old.cs
using System;

public class void Main(string[] args){
    //request name
    Console.Write("What is your name? ");
    string user_name = Console.ReadLine();

    Console.Write("Enter a number: ");
    int repetition = int.Parse(Console.ReadLine());

    name_print(user_name, repetition);
}

public class string name_print(string name, int repeat){
    while(repeat>0){
        Console.Write(name);
        repeat--;
    }
    
}

File: Python/Practice/Mother of A 3-year-old.py
# Request the user's name and their choice of how many times they want their name printed


def name_print(name, print_num):

    run = True
    while (run):
        print(user_name)
        print_num-=1
        if print_num == 0:
            run = False


user_name = input("Enter your name: ")
repeat = int(input("Repeat how many times? "))
name_print(user_name, repeat)
